src/components/evaluator/services/znkEvaluatorSrv.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+(function (angular) {
+    angular.module('znk.infra.evaluator').provider('ZnkEvaluatorSrv', function () {
+
+        var _evaluateQuestionGetter;
+
+        this.shouldEvaluateQuestion = function(evaluateQuestionGetter) {
+            _evaluateQuestionGetter = evaluateQuestionGetter;
+        };
+
+        this.$get = function ($log, $q, $injector, ENV, $http, InfraConfigSrv) {
+            'ngInject';
+
+            var znkEvaluatorSrvApi = {};
+
+            var httpConfig = {
+                timeout: ENV.promiseTimeOut
+            };
+
+            function _shouldEvaluateQuestion() {
+                if(!_evaluateQuestionGetter){
```

Comment:
the default one is loaded through config phase, didn't want to add dependency in purchaseService